### PR TITLE
fixes broken marked one spin attack and proc and removes crusher from arena

### PR DESCRIPTION
**Pull request by dawsonkeyes**
## About The Pull Request

This make’s the marked one’s spin attack actually work (sleep_check_death moment), makes the wendigo stomp proc indigenous to the marked one’s code (aka fixes #17055 linters), and removes the kinetic crusher from the lavaland arena

## How This Contributes To The Skyrat Roleplay Experience

The spin attack should work, the linters shouldn’t be broken, and the ash walkers shouldn’t have near-roundstart access to miner tech weaponry.

I didn’t PR these changes individually because that would only serve to farm gbp like a dickhead, and none of them are large enough to generally warrant individual pull requests.

## Changelog

:cl:
balance: kinetic crusher no longer spawns in the marked ones arena 
fix: the marked ones spin attack is no longer faulty, he is now a beyblade again  
/:cl:

### CODE REVIEW

**Comment by dawsonkeyes** [comment]
> You currently have a negative Fix/Feature pull request delta of -8. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

AND THIS
IS TO GO
EVEN
FURTHER
BEYOND

**Review by Paxilmaniac — APPROVED** [review]
image

File: modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm [hunk 1]
```diff
@@ -376,11 +372,33 @@
 	boned.throw_at(target, 7, 3, thrower = src)
 	QDEL_IN(boned, 3 SECONDS)
 
+/mob/living/simple_animal/hostile/megafauna/gladiator/proc/ctrl_c_ctrl_v(range, delay, throw_range) //copied and pasted wendigo slam attack, used for more slowly radiating AOE slams
```
Comment (larentoun) [inline modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm]: `ctrl_c_ctrl_v` ???

Comment (dawsonkeyes) [reply modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm]: see the comment
yes it’s a shit in-joke and I’ll change it

File: modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm [hunk 2]
```diff
@@ -322,13 +322,9 @@
...
 			break
-		addtimer(CALLBACK(src, .proc/animate_speen), 0.5 SECONDS)
-
-/mob/living/simple_animal/hostile/megafauna/gladiator/proc/animate_speen()
+		sleep(0.75)
 	animate(src, color = initial(color), 3)
-	addtimer(CALLBACK(src, .proc/stop_speen), 0.5 SECONDS)
-
-/mob/living/simple_animal/hostile/megafauna/gladiator/proc/stop_speen()
+	sleep(3)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm:336]: I really don't like the usage of sleeps, I guess we won't have a choice here, eugh.

File: modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm [hunk 3]
```diff
@@ -376,11 +372,33 @@
...
+/mob/living/simple_animal/hostile/megafauna/gladiator/proc/ground_pound(range, delay, throw_range) //copied and pasted wendigo slam attack, used for more slowly radiating AOE slams 
+	var/turf/origin = get_turf(src)
+	if(!origin)
+		return
+	var/list/all_turfs = RANGE_TURFS(range, origin)
+	for(var/sound_range = 0 to range)
+		playsound(origin,'sound/effects/bamf.ogg', 600, TRUE, 10)
+		for(var/turf/stomp_turf in all_turfs)
+			if(get_dist(origin, stomp_turf) > sound_range)
+				continue
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm:398]:
```suggestion
				continue

```

File: modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm [hunk 4]
```diff
@@ -376,11 +372,33 @@
...
+			new /obj/effect/temp_visual/small_smoke/halfsecond(stomp_turf)
+			for(var/mob/living/target in stomp_turf)
+				if(target == src || target.throwing)
+					continue
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm:403]:
```suggestion
					continue

```

File: modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm [hunk 5]
```diff
@@ -376,11 +372,33 @@
...
+				to_chat(target, span_userdanger("[src]'s ground slam shockwave sends you flying!"))
+				var/turf/thrownat = get_ranged_target_turf_direct(src, target, throw_range, rand(-10, 10))
+				target.throw_at(thrownat, 8, 2, null, TRUE, force = MOVE_FORCE_OVERPOWERING, gentle = TRUE)
+				target.apply_damage(20, BRUTE, wound_bonus=CANT_WOUND)
+				shake_camera(target, 2, 1)
+			all_turfs -= stomp_turf
+		sleep(delay)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm]:
```suggestion

				shake_camera(target, 2, 1)

			all_turfs -= stomp_turf

		sleep(delay)
```

File: modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm [hunk 6]
```diff
@@ -376,11 +372,33 @@
...
+
+/mob/living/simple_animal/hostile/megafauna/gladiator/proc/swordslam() //large radius but slow-to-move radiating ground slam
+	ground_pound(5, 1 SECONDS, 8)
 
-/mob/living/simple_animal/hostile/megafauna/gladiator/proc/stomp()
-	wendigo_slam(src, 2, 0.5 SECONDS, 3)
+/mob/living/simple_animal/hostile/megafauna/gladiator/proc/stomp() //short range slam with faster shockwave travel
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm]: documentation please (`/// documentation here` on the line before your proc (still keep at least one empty newline between each proc))

**Review by GoldenAlpharex — DISMISSED** [review]

**Review by Zonespace27 — APPROVED** [review]

**Review by tf-4 — APPROVED** [review]

**Pull request merged**